neuron/src/app/Neuron/CLI/Types.hs

```diff
@@ -46,7 +46,7 @@ data App = App
 data NewCommand = NewCommand
   { title :: Maybe Text,
     format :: Maybe ZettelFormat,
-    day :: Day,
+    localtime :: LocalTime,
```

Comment:
Call this `date :: LocalTime`. Since that's what it is called in the markdown metadata.